### ISSUE TITLE
MultiSelect: Trigger lacks `role` and `aria-haspopup` attributes

### ISSUE THREAD

**Issue by illfygli** [primefaces/primeng#11894]
### Describe the bug

I am using a keyboard-oriented browser as assistive technology, and cannot activate the MultiSelect component.

The Dropdown component has `role="button"` on its "trigger" element, which lets me activate it, so it would be nice if the MultiSelect component also had it.

The issue is visible in the [component gallery](https://www.primefaces.org/primeng/multiselect), where none of the examples can be activated with such a browser.

### Environment

[Qutebrowser](https://qutebrowser.org/) on Linux.

### Reproducer

https://www.primefaces.org/primeng/multiselect

### Angular version

13.3.8

### PrimeNG version

13.3.2

### Build / Runtime

TypeScript

### Language

TypeScript

### Node version (for AoT issues node --version)

14.20.0

### Browser(s)

qutebrowser, vimb

### Steps to reproduce the behavior

1. Go to the [MultiSelect component demo page](https://www.primefaces.org/primeng/multiselect) using a keyboard-oriented browser.
2. Try to activate any of the MultiSelects.

### Expected behavior

It should be possible to activate a MultiSelect using a keyboard-oriented browser, in the same way that the [Dropdown component](https://www.primefaces.org/primeng/dropdown) allows.

**Comment by illfygli** [comment]
Here's what it's like with the MultiSelect, where there's nothing I can activate:
![screen](https://user-images.githubusercontent.com/42881386/188119219-754915e5-48a7-45e8-9a45-e35130733bda.png)

And here's the Dropdown, which works fine:
![dropdown](https://user-images.githubusercontent.com/42881386/188119278-242e0438-5aa0-4bdc-9b91-85ec27a7715b.png)

**Comment by illfygli** [comment]
Radio buttons and checkboxes have the same problem.

Can be seen/reproduced [on the PrimeFaces website](https://www.primefaces.org/primeng/).
